openapi/SwarmDebug.yaml

```diff
@@ -236,7 +236,7 @@ paths:
           required: true
           description: Underlay address of peer
       responses:
-        "200":
+        "201":
```

Comment:
this should stay 200, since when you call the API endpoint, it returns when you connect to the peer, so it is OK rather than Created (which might infer the operation not still complete)